packages/cli/fragments/fragment-vanilla-ts/src/main.ts

```diff
@@ -0,0 +1,25 @@
+import { invoke } from "@tauri-apps/api";
```

Comment:
use global tauri instead. `window.__TAURI__`

```suggestion
const { invoke } = window.__TAURI__;
```